Demos/Blazorise.Demo/Pages/Tests/ChartsDataLabelsPage.razor.cs

```diff
@@ -182,7 +182,8 @@ public partial class ChartsDataLabelsPage
                 BackgroundColor = BackgroundColors[2],
                 BorderColor = BorderColors[2],
                 Align = "center",
-                Anchor = "center"
+                Anchor = "center",
+                SoftCodeFormatter = "function(value, context) { return \"$\" + value; }"
```

Comment:
I tried using the `scriptableOption` method but using this method allows us to soft code the formatter logic and make it as complex as we want.